MakingMusicTool/MakingMusicTool.py

```diff
@@ -0,0 +1,363 @@
+# @Author  : Tom Brandherm (tom.brandherm@msasafety.com)
+# @Link    : link
+# @Version : 1.0.0
+"""
+Making music with python is possible by using this tool. As a import pattern 
+please use a created csv file with one or more rows like:
+
+[one of the given instruments],[volume from 1 to 3],[music note 1],...[music note n]
+
+A 4/4 time is used here, so the sum of the notes in one row have to be equals one.
+The notation for the notes into the csv is: 
+
+1/16    =>  16
+1/8     =>  8
+1/4     =>  4
+1/2     =>  2
+1/1     =>  1
+"""
+# =========================================================================== #
+#  Copyright 2021 Team Awesome
+# =========================================================================== #
+#  All Rights Reserved.
+#  The information contained herein is confidential property of Team Awesome.
+#  The use, copying, transfer or disclosure of such information is prohibited
+#  except by express written agreement with Team Awesome.
+# =========================================================================== #
+
+# =========================================================================== #
+#  SECTION: Imports                                                           
+# =========================================================================== #
+import os
+import csv
+
+from pydub import AudioSegment
+from pydub.playback import play
+# =========================================================================== #
+#  SECTION: Global definitions
+# =========================================================================== #
+# name of the input csv file
+FILENAME = "sound_pattern.csv"
+# =========================================================================== #
+#  SECTION: Class definitions
+# =========================================================================== #
+
+
+class MusicMaker(object):
+    """
+    MusicMaker is a class to that can create soundtracks out of a given pattern.
+    The pattern needs information about a instrument, the volume and a 4/4-time
+    rythm.
+    """
+    # ----------------------------------------------------------------------- #
+    #  SUBSECTION: Constructor
+    # ----------------------------------------------------------------------- #
+
+    def __init__(self, fileName: str, bpm:int=60):
+        ## public
+        # input from the csv file
+        self.sound_pattern = self.analyse_data(fileName)
+        ## __private
+        # given instrument beat audio files (wav-format!!!)
+        self.__instruments =   {'drum': 'mixkit-metal-hit-drum-sound-550.wav',
+                                'bass': 'mixkit-bass-guitar-single-note-2331.wav',
+                                'tribal drum': 'mixkit-tribal-dry-drum-558.wav',
+                                'violin': 'mixkit-orchestral-violin-jingle-2280.wav'}
+        # convert the beats per minutes (bpm) into the duration of one 4/4 time [ms]
+        self.__44time_duration = 60*4/bpm*1000
```

Comment:
Amelie thought it read as 44 instead of 4/4. questions is if we need 4/4 really in the name.
i know we only accept 4/4, but could be more modular by having the time signature as a parameter in the suggested method. than the name could be duration or time_signature_duration

```suggestion
        self.__4_4time_duration = convert_bpm_to_4_4_time_duration(bpm)
```
or
```suggestion
        self.__time_signature_duration = convert_bpm_to_time_signature_duration_in_ms(bpm, timeSignature)
```

if changed the comment above becomes unnecessary